addons/nightvision/functions/fnc_pfeh.sqf

```diff
@@ -85,6 +85,10 @@ if (CBA_missionTime < GVAR(nextEffectsUpdate)) then {
     // Modify blur if looking down scope
     if ((cameraView == "GUNNER") && {[ACE_player] call CBA_fnc_canUseWeapon}) then {
         if (currentWeapon ACE_player == "") exitWith {};
+        if (
+            currentWeapon ACE_player == primaryWeapon ACE_player && 
+            count ("'NVG' in getArray (_x >> 'visionMode')" configClasses (configFile >> "CfgWeapons" >> ((primaryWeaponItems ACE_player) select 2) >> "ItemInfo" >> "OpticsModes")) > 0
+        ) exitWith {_blurFinal = _blurFinal * linearConversion [0, 1, GVAR(aimDownSightsBlur), 1, ST_NVG_CAMERA_BLUR_SIGHTS_NVS]}; // NVS are good
```

Comment:
```sqf
#define ST_NVG_CAMERA_BLUR_SIGHTS_NVS 1
linearConversion [0, 1, GVAR(aimDownSightsBlur), 1, ST_NVG_CAMERA_BLUR_SIGHTS_NVS]
```
`linearConversion` from 1 to 1 makes no sense.

Reply:
Yes absolutely, I think there is no need to update the `_blurFinal` value.